### ISSUE TITLE
Replace Invalid Household ID (0 or -1) with a missing value

### ISSUE THREAD

**Issue by annaschultze** [opensafely-core/cohort-extractor#446]
At the moment household_ID can generate invalid values for when someone's address or postcode is missing. As this is not clearly indicated in the expectations framework (my fault), I hadn't expected this and accidentally included those with missing household ID (who were coded as -1 or 0) as a category in itself. 

Although this quickly became obvious on inspection of the data, it might be easier to just ensure that whenever address is missing it doesn't add any value at all. 

This may be covered in docs already but seemed like it could be a potentially simple fix that could avoid unintentional errors due to unexpected missing data.

**Comment by esnightingale** [comment]
Am I right in thinking household size is also 0 if missing?

**Comment by evansd** [comment]
@esnightingale Yes, that's correct. We're discussing changing all this behaviour in the linked ticket above.
